test/data/mini_example/insertions_output.fasta

```diff
@@ -0,0 +1,40 @@
+> allele_0
```

Comment:
After we merge #82, we don't need this file anymore.

Reply:
jepp :)